syntax = "proto3";
import "google/protobuf/empty.proto";
import "google/protobuf/timestamp.proto";

message RawMeteoData2 {
  string id = 1;
  float temperature = 2;
  float humidity = 3;
  google.protobuf.Timestamp timestamp = 4;
}

message RawPollutionData2 {
  string id = 1;
  float co2 = 2;
  google.protobuf.Timestamp timestamp = 3;
}

service ComputeServerService {
  rpc process_meteo_data (RawMeteoData2) returns (google.protobuf.Empty) {}
  rpc process_pollution_data (RawPollutionData2) returns (google.protobuf.Empty) {}
}